tests/test_bulk.py

```diff
@@ -13,6 +13,21 @@ async def test_bulk_create(self):
         inc = all_[0]["id"]
         self.assertEqual(all_, [{"id": val + inc, "name": None} for val in range(1000)])
 
+    async def test_bulk_create_with_specified(self):
+        await UniqueName.bulk_create([UniqueName(id=id_) for id_ in range(1000)])
```

Comment:
I think I have an idea re PostgreSQL. The id you generate is from 0 to 999
I don't think 0 is a safe serial number. It should start from 1 onwards?

Reply:
Good. I'll Change range value and will retry it